.github/workflows/pychecks.yaml

```diff
@@ -0,0 +1,32 @@
+name: Python Checks
+
+on:
+  push:
+    branches: [staging, stable]
+    paths:
+      - 'support/scripts/**'
+      - '.github/workflows/pychecks.yaml'
+
+  pull_request:
+    types: [opened, synchronize, reopened]
+    branches: [staging]
+    paths:
+      - 'support/scripts/**'
+      - '.github/workflows/pychecks.yaml'
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: black
+        uses: psf/black@stable
+        with:
+          src: support/scripts
+
+      - name: ruff
+        uses: chartboost/ruff-action@v1
+        if: success() || failure()
```

Comment:
What about running this in a different job instead?

Reply:
We would need to double-checkout, but the advantage is that they will run in parallel (+ show up separately)

Reply:
I'm fine either way